www/js/app.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+// 'weather' is referenced in index.html, 2nd arg is dependencies
```

Comment:
As your directives, controllers, etc. get larger it might make sense to split them into multiple files. But I think a header comment for each of them here would still be helpful.